### PR TITLE
Add `mainnet`, `testnet`, `betanet`; remove `exit-precompiles` compilation features

### PR DESCRIPTION
**Pull request by sept-en**
The purpose of PR is to be able to always build for a specific network (Mainnet/Testnet/Betanet) with explicitly defined set of features.

Please also note that `--always-make` or `-B` parameter to `make` (specified in `README.md`) is mandatory, otherwise `Makefile` won't rebuild with just only a different feature.

The expected usage is:

```sh
make --always-make mainnet # Build for  Mainnet
make --always-make testnet # Build for Testnet
make --always-make betanet # Build for Betanet
```

### CODE REVIEW

**Review by joshuajbouw — APPROVED** [review]
Also, straight forward.

**Comment by artob** [comment]
@sept-en I think you can go ahead and remove the `exit-precompiles` flag while we're at it, I don't see why we wouldn't build with it always going forward now.

File: README.md [hunk 1]
```diff
@@ -52,6 +52,9 @@ rustup target add wasm32-unknown-unknown --toolchain nightly-2021-03-25
 ### Building the EVM binary
 
 ```sh
+make --always-make mainnet # produces Mainnet build
+make --always-make testnet # produces Testnet build
+make --always-make betanet # produces Betanet build
```
Comment (artob) [inline README.md:57]: Leave two spaces prior to the '#' character.

**Review by birchmd — APPROVED** [review]
Nice!

**Review by artob — APPROVED** [review]

**Pull request merged**